week5/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM python:latest as base
```

Comment:
`python:latest` -> let's use defined version of python instead latest

Reply:
fixed